ide/diff/src/org/netbeans/modules/diff/builtin/visualizer/editable/DiffContentPanel.java

```diff
@@ -67,6 +67,8 @@ public DiffContentPanel(EditableDiffView master, boolean isFirst) {
         add(scrollPane);
         
         linesActions = new LineNumbersActionsBar(this, master.isActionsEnabled());
+        linesActions.addMouseWheelListener(e -> editorPane.dispatchEvent(e)); // delegate mouse scroll events to editor
+
```

Comment:
should we do this instead?
```java
        linesActions.addMouseWheelListener(e -> {
            editorPane.dispatchEvent(e); // delegate mouse scroll events to editor
            e.consume();
        });
```
i am slightly worried that this is a JDK bug, since this did already stop the scrolling:
```java
        linesActions.addMouseWheelListener(e -> {});
```

Reply:
Either is fine with me, as long as it works on all platforms.

Reply:
I tested it on linux